Trimble.Modus.Components/Controls/TMChips/TMChips.xaml

```diff
@@ -3,9 +3,22 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
              HorizontalOptions="Start"
              VerticalOptions="Start"
+             x:Name="tmChip"
+             xmlns:local="clr-namespace:Trimble.Modus.Components"
              xmlns:constants="clr-namespace:Trimble.Modus.Components.Constant"
+             xmlns:converters="clr-namespace:Trimble.Modus.Components.Converters"
              x:Class="Trimble.Modus.Components.TMChips">
-  <Border x:Name="frame"

```

Comment:
Icon colors should change along with the text color in dark theme (i.e should be in white)

![Simulator Screen Shot - iPhone 14 - 2023-12-01 at 16 45 06](https://github.com/trimble-oss/modus-mobile-maui-components/assets/60956922/1b6475ab-d4c2-4ad8-a162-9d7afd50014f)